//
// Created by zackliu on 4/4/17.
//

#ifndef CLOUDSTORAGE_BLOCK_MANAGER_H
#define CLOUDSTORAGE_BLOCK_MANAGER_H

#include "../proto/chunkserver.pb.h"
#include "../proto/nameserver.pb.h"
#include "../proto/status_code.pb.h"

#include <common/thread_pool.h>

#include "counter_manager.h"



#endif //CLOUDSTORAGE_BLOCK_MANAGER_H
